src/Helper/Product/Stock/Utility.php

```diff
@@ -30,8 +30,8 @@
 use Magento\CatalogInventory\Model\ResourceModel\Stock\Item as StockItemResource;
 use Magento\CatalogInventory\Model\Stock;
 use Magento\Framework\App\ObjectManager;
-use Magento\Framework\App\ProductMetadataInterface;
 use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Module\Manager as ModuleManager;
 use Magento\InventoryCatalog\Model\GetStockIdForCurrentWebsite;
 use Magento\InventoryExportStock\Model\GetStockItemConfiguration;
 use Magento\InventorySalesApi\Model\GetStockItemDataInterface;
```

Comment:
This use was causing issues on instances without MSI:

> <b>Fatal error</b>: Uncaught Error: Cannot instantiate interface Magento\InventorySalesApi\Model\GetStockItemDataInterface in /home/webuser/websites/franksgreatoutdoors.com/public_html/releases/20200402135740/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php:121

It was not just about if module was disabled. It is completely missing in code on affected instances.

Reply:
I also just deleted the magento/module-inventory-sales-api on my instance + setup:di:compile. It is still working.
The error is indicating that magento could not instantiate an object that is implementing the interface because there is no preference for interface defined if the sales-api module is missing. The use statement itself is not harmful as it does not load or include a physical file, but indeed ugly.

I could switch to fully qualified names and therefor remove the use statements, e.g.:
$getStockItemData = $objectManager->get(\Magento\InventorySalesApi\Model\GetStockItemDataInterface::class);

Reply:
Okay cool. The important thing is that it will work for that scenario. 
For the visual polish it does not matter if its an not existing use or fully qualified name in call i guess. So we can leave it as it is. Thank you!